src/mscorlib/Resources/Strings.resx

```diff
@@ -589,11 +586,11 @@
   <data name="Arg_ParmCnt" xml:space="preserve">
     <value>Parameter count mismatch.</value>
   </data>
-  <data name="Arg_PathIllegal" xml:space="preserve">
-    <value>The path is not of a legal form.</value>
+  <data name="Arg_PathEmpty" xml:space="preserve">
```

Comment:
Did you double check these strings to see if they already exist (like in desktop mscorlib)? Some may have gotten dropped in all of the code moves.